src/styles/index.scss

```diff
@@ -2,2 +1,237 @@
+.page {
   margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+:root {
+  --main-text-color: #00acdc;
+}
+
+.header {
+  margin: 0;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  background-color: #fff;
+  justify-content: space-between;
```

Comment:
The properties `justify-content: space-between;` and `align-items: center;` have no effect here because they are used without a display property. You should add a `display: flex;` or `display: grid;` property to the .header class.